packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -117,6 +117,18 @@ export class Loan {
     return await this.client.call('listcollateraltokens', [], 'bignumber')
   }
 
+  /**
+   * Get collateral token.
+   *
+   * @param {GetCollateralToken} [collateralToken = {}]
+   * @param {string} [collateralToken.token] Symbol or id of collateral token
+   * @param {number} [collateralToken.height] Valid at specified height
```

Comment:
```suggestion
   * @param {number} [collateralToken.height] height filter, activation height less than or equal to
```

Reply:
The text is copied from ain. I stick to `Valid at specified height`. If want to change, the text should be height filter, `activation height more than or equal to`

Reply:
looking at c++ side implementation looks like `height` has a default value, which is the current block height. so if a  height is not given, the default value will be used. I'm thinking if we should reflect that in the documentation🤔 or just keep it as it is. 
`current block height` is not a constant value, Then how should we do it?
`@param {number} [collateralToken.height = current block height]`
@fuxingloh

Reply:
Ref - https://github.com/DeFiCh/ain/blob/3b17b2ede136fa1931e2831415ddf7e5f4a15a22/src/masternodes/rpc_loan.cpp#L187

Reply:
Or set the default value for height in Javascript side?
https://github.com/DeFiCh/jellyfish/pull/568#discussion_r705105016

Reply:
> Or set the default value for height in Javascript side?
> [#568 (comment)](https://github.com/DeFiCh/jellyfish/pull/568#discussion_r705105016)

we can't really set a default value for this since it's stateful dependent on current block height,

```
@param {number} [collateralToken.height = CurrentBlockheight]
```

as @surangap comment, the above works

Reply:
Fixed